joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/ast/KtPsiToAst.scala

```diff
@@ -249,8 +254,8 @@ trait KtPsiToAst {
 
     val typeFullName = typeInfoProvider.typeFullName(ktClass.getPrimaryConstructor, TypeConstants.any)
```

Comment:
seems we could delete this line now

Reply:
ugh, thx, missed it, i'll bring the removal in in a next pr